src/main/java/net/sf/jabref/exporter/CustomExportList.java

```diff
@@ -69,12 +71,12 @@ private void readPrefs() {
         formats.clear();
         list.clear();
         int i = 0;
-        String[] s;
-        while ((s = Globals.prefs.getStringArray(JabRefPreferences.CUSTOM_EXPORT_FORMAT + i)) != null) {
+        List<String> s;
+        while (!((s = Globals.prefs.getStringList(JabRefPreferences.CUSTOM_EXPORT_FORMAT + i)).isEmpty())) {
             Optional<ExportFormat> format = createFormat(s);
             if (format.isPresent()) {
                 formats.put(format.get().getConsoleName(), format.get());
-                list.add(s);
+                list.add(s.toArray(new String[s.size()]));
```

Comment:
I do not think so. I do not want to add the elements but the array. `EventList<String[]>`. Or am I missing something?